pipeline/ref_maker.py

```diff
@@ -481,26 +486,197 @@ def make_refset(self, session=None):
+        if calling this from the outside.)
+
+        Returns
+        -------
+           images, match_pos, match_count
+
+           images: list of Image
+             Images that can be included in the sum
+
+           match_pos: 2d numpy array
+             Each row is [ra,dec] of a position on the summed image.  If
+             operating in (ra,dec) mode (rather than min/max ra/dec
+             mode), this will be [[ra,dec]].
+
+          match_count: list of int
+             Number of images that overlap the corresponding match_pos.
+
+        """
+        if not _do_not_parse_arguments:
+            self.parse_arguments( *args, **kwargs )
+
+        if self.pars.corner_distance is None:
+            match_pos = [ [ self.ra, self.dec ] ]
+            match_count = [ 0 ]
+            kwargs = { 'ra': self.ra, 'dec': self.dec }
+        else:
+            if ( self.maxra < self.minra ):
+                dra = ( self.maxra + 360. - self.minra ) * self.pars.corner_distance/2.
+                ctrra = ( self.maxra+360. + self.minra ) / 2.
+                ctrra = ctrra if ctrra >= 0. else ctrra + 360.
+            else:
+                dra = ( self.maxra - self.minra ) * self.pars.corner_distance/2.
+                ctrra = ( self.maxra + self.minra ) / 2.
+            ddec = ( self.maxdec - self.mindec ) * self.pars.corner_distance/2.
+            ctrdec = ( self.maxdec + self.mindec ) / 2.
+            match_pos = np.array( [ [ ctrra + 0.,  ctrdec + 0. ],
+                                    [ ctrra - dra, ctrdec - ddec ],
+                                    [ ctrra + 0.,  ctrdec - ddec ],
+                                    [ ctrra + dra, ctrdec - ddec ],
+                                    [ ctrra - dra, ctrdec + 0. ],
+                                    [ ctrra + dra, ctrdec + 0. ],
+                                    [ ctrra - dra, ctrdec + ddec ],
+                                    [ ctrra + 0.,  ctrdec + ddec ],
+                                    [ ctrra + dra, ctrdec + ddec ] ] )
+            match_count = [ 0 ] * 9
+            kwargs = { 'minra': self.minra, 'maxra': self.maxra, 'mindec': self.mindec, 'maxdec': self.maxdec,
+                       'overlapfrac': self.pars.coadd_overlap_fraction }
+
+        kwargs['provenance_ids'] = [ p.id for p in self.im_provs.values() ]
+        kwargs['instrument' ] = self.pars.instruments
+        kwargs['project'] = self.pars.projects
+        kwargs['filter'] = self.filter
+        kwargs['min_mjd'] = None if self.pars.start_time is None else parse_dateobs( self.pars.start_time, output='mjd' )
+        kwargs['max_mjd'] = None if self.pars.end_time is None else parse_dateobs( self.pars.end_time, output='mjd' )
+        kwargs['max_seeing'] = self.pars.max_seeing
+        kwargs['min_lim_mag'] = self.pars.min_lim_mag
+        kwargs['min_exp_time'] = self.pars.min_exp_time
+        # TODO : airmass, background
```

Comment:
Yeah.  I don't anticipate actually ever searching on airmass or background, in reality, because the practical effects of high airmass and high background is that the seeing and limiting magnitudes get worse, so we're more likely to just search on that.